app/views/lifehacks/_lifehack.html.erb

```diff
@@ -1,22 +1,31 @@
+
 <div class="row">
-<!-- Figure out to nest this "fixed elemnt projfsperly" -->
-  <div class="large-8 columns  clearfix">
-    <div class="panel right large-12 columns">
-      <h5><%= link_to lifehack.title, lifehack_path(lifehack) %></h5>
-      <table>
+    <% @lifehacks.each do |lifehack| %>
```

Comment:
if we do this, won't we end up putting out everything multiple times? Since we are calling `render @lifehacks` I believe it is already iterating through the collection.